system-int-test/test/cmr/system_int_test/bootstrap/reshard_index_test.clj

```diff
@@ -0,0 +1,69 @@
+(ns cmr.system-int-test.bootstrap.reshard-index-test
+  "Tests rebalancing granule indexes by moving collections's granules from the small collections
+   index to separate collection indexes"
+  (:require
+   [clojure.test :refer [deftest is testing use-fixtures]]
+   [cmr.system-int-test.data2.collection :as dc]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.granule :as dg]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.bootstrap-util :as bootstrap]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"
+                                           "provguid2" "PROV2"}))
+
+;; These tests don't actually require Oracle, but they are marked as such to keep them from
+;; running normally as they can be slow
```

Comment:
Is this comment correct or is it because bootstrap does not work with an in-memory database? If this comment is true that seems like a bad idea - we shouldn't be marking things as only with real database as some kind of proxy on how fast the test is - if we want to do something like that it should be a different flag.

Reply:
If it is a needed test, even if it is slow, shouldn't it always run?  Developers won't know to specifically run this test even if they change something.  Then they will only find out if there is a problem when bamboo runs this test.